services/database-analyzer.js

```diff
@@ -17,6 +17,33 @@ function DatabaseAnalyzer(databaseConnection, config) {
     return Object.keys(schema).filter(column => schema[column].primaryKey);
   }
 
+  function isUnderscored(fields) {
+    let underscored = false;
```

Comment:
we could simplify this like that:
`return fields.some(field => field.name.includes('_'));`